serverless/internal/storage/fs/fs.go

```diff
@@ -107,7 +107,7 @@ func Create(rootDir string) (*Storage, error) {
 // Returns the sequence number assigned to this leaf (if the leaf has already
 // been sequenced it will return the original sequence number and
 // storage.ErrDupeLeaf).
-func (fs *Storage) Sequence(leafhash []byte, leaf []byte) (uint64, error) {
+func (fs *Storage) Sequence(ctx context.Context, leafhash []byte, leaf []byte) (uint64, error) {
```

Comment:
I think any of these `ctx` params that are not used in their respective function bodies can be replaced `_`?